packages/react/src/search/Search.tsx

```diff
@@ -0,0 +1,58 @@
+import { useComposedRefs } from "@radix-ui/react-compose-refs";
+import { useControllableState } from "@radix-ui/react-use-controllable-state";
+import { type ComponentPropsWithRef, forwardRef, useRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Button } from "../button";
+import { Input } from "../input";
+import { IconCross } from "./IconCross";
+import { IconSearch } from "./IconSearch";
+
+type SearchProps = ExtendProps<
+  ComponentPropsWithRef<typeof Box>,
+  Omit<ComponentPropsWithRef<typeof Input>, "onChange">,
+  {
+    defaultValue?: string;
+    onChange?: (value: string) => void;
+    value?: string;
+  }
+>;
```

Comment:
we can simplify this
```suggestion
type SearchProps = ComponentPropsWithRef<typeof Input>;
```